lib/dyte/resources/meetings.rb

```diff
@@ -33,5 +33,9 @@ def fetch_participants(meeting_id:)
     def delete_participant(meeting_id:, participant_id:)
       delete_request("meetings/#{meeting_id}/participants/#{participant_id}")
     end
+
+    def edit_participant_details(meeting_id:, participant_id:, attributes:)
+      patch_request("meetings/#{meeting_id}/participants/#{participant_id}", body: attributes)
```

Comment:
```suggestion
      Participant.new patch_request("meetings/#{meeting_id}/participants/#{participant_id}", body: attributes)
```

Let's wrap this in a Participant class.